templates/saga/index.ts.mustache

```diff
@@ -0,0 +1,31 @@
+import { createAction, createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+// TODO: Add reducer to ../reducer.ts
+// TODO: Add saga to ../saga.ts
+
+export enum SagaActionTypes {
+  Start = '{{sagaName}}/start',
+}
+
+export type {{sagaName}}State = {
```

Comment:
just wondering, is it possible to pass the `sagaName` & `component name` from cli arguments (optional args)?

Reply:
Those get calculated based on the folder name.

`node ./templates/generate.js component src/components/funny-button` will create a component named `FunnyButton`, for example.